CONTRIBUTING.md

```diff
@@ -0,0 +1,261 @@
+# Contributing to terraform-provider-aap
+
+Thank you for contributing to the Ansible Automation Platform Terraform Provider!
+
+These guidelines document the standards the Inframesh team follows and holds each other accountable to. Whether you're an internal team member or external contributor, following these guidelines ensures consistent, high-quality contributions.
+
+## Quick Start
+
+| Step | Command | Purpose |
+|------|---------|---------|
+| 1. Fork & clone | `git clone https://github.com/<your-username>/terraform-provider-aap` | Get the code |
+| 2. Choose base branch | `git checkout main` or `git checkout dev-preview` | See [Branch Strategy](#branch-strategy) |
```

Comment:
Branches should be created from `main`, see https://github.com/ansible/terraform-provider-aap/blob/main/README.md#branching-model.